README.md

```diff
@@ -175,3 +175,23 @@ This manager makes it easier to mount ssh drives using the "fusermount" command.
  * 8) Exit                            *
  **************************************
 ```
+# Git-Create
+A package that can add/remove/edit remote repositories in GitHub from the terminal
+
+## Branches: 
+- Master Branch : Contains files to be downloaded.
+- Installed Branch : Contains files after download and installation (ie.. the entire project).
+
+## Installation: 
+- Requires Python, Selenium , Geckodriver to be installed.
+- Run the installation.sh file.
+- Enter UserName and Password when prompted.
+- It will setup all the required files.
+
+## Usage:
+- Go to the directory where you want to setup your git.
+- Type 'gitc.sh' in cmd, the program will start.
```

Comment:
where is this script folder? Is it in the git-shell module? If so, how do I view the files in the submodule, I have never used them before.